internal/uvm/create.go

```diff
@@ -160,6 +163,9 @@ func verifyOptions(ctx context.Context, options interface{}) error {
 		if len(opts.LayerFolders) < 2 {
 			return errors.New("at least 2 LayerFolders must be supplied")
 		}
+		if opts.SCSIControllerCount < 1 {
```

Comment:
Should we instead be checking for `SCSIControllerCount != 1`? Does WCOW support multiple controllers? If so can we add a test for it?

Reply:
Changed this to `SCSIControllerCount != 1` since we haven't tested the multiple SCSI controller support for WCOW.